content/blog/cookie-consent-banner.mdx

```diff
@@ -0,0 +1,102 @@
+    engineers creating those. An awesome developer experience leads to an awesome
+    (end-) user experience. That's why I enjoy spending time into defining a
+    meaningful and scalable interface definition. The Cookie Consent Banner is such
+    an example. It's a component that you have to build for legal reasons and
+    probably don't touch quite often after the initial implementation. It shouldn't
+    be complicated if you need to adapt it again after some time. The Cookie Consent
+    Banner is very flexible, but still convenient to integrate.  
+
+</ImageText>
+
+<ImageText imageSrc="/assets/blog/ccb/1.png" imageAlt="Collection of websites using the cookie banner" theme="light" left
+    title="You are currently maintaining Porsche's open-source project Cookie Consent Banner on GitHub. How would you explain your project to a five-year-old? What problem does your project solve for Porsche?">
+
+    **Patrick:** I'm sure everybody remembers the time when cookie banners hit the
+    internet. It's not only cumbersome for the visitors of a website but also for
+    the maintainers to implement such a solution. So we created a component that is not
+    only informative to the visitors of a website, but also easy to integrate for
+    the product teams. It should be as straightforward as possible to integrate
+    third-party scripts and cookies into a website in a privacy-friendly manner,
+    always respecting the decision of the visitor. That's why the component doesn't
+    do any magical gatekeeping of such scripts, but instead making it very
+    convenient for the developer to get to know the information about the consent of
+    the visitor. The developers should have total control over what's happening on the
+    website and which scripts are being loaded.  
+    <br/>
+    **Tim:** In innovation projects, we work in a particularly agile manner and in
+    short development cycles. A cookie banner was therefore the perfect opportunity
+    to give something back to the community and at the same time create a building
+    block that enables innovation projects to build GDPR-compliant pages with
+    minimal effort.
+</ImageText>
+
+<ImageText imageSrc="/assets/blog/ccb/2.jpg" imageAlt="Chart showing weekly downloads of the Cookie Consent Banner" theme="light"
+    title="With over 3000 weekly downloads your project already gained some adoption among the community. What did you learn along the way of maintaining an open-source project?">
+
+    **Patrick:** The component should be easy to use, thus we need to ensure a
+    balance between sensible defaults and a potentially overwhelming set of
+    options to customize. Given the success of the component, I would say we
+    found a good balance. I really like the feedback the community provides via
+    GitHub Issues and Pull Requests. For example, did we face an issue where
+    someone mentioned how the component affects their Google Lighthouse Score.
```

Comment:
Replace phrase with "For example, one of the issues we had was someone mentioning how the component was affecting their Google Lighthouse score."